### PR TITLE
code style fixes for QNX debugger

### PR DESCRIPTION
**Pull request by madprogrammer**
Code style fixes for QNX debugger plugin, see https://github.com/radare/radare2/pull/4678 for details

### CODE REVIEW

File: shlr/qnx/include/arch.h [hunk 1]
```diff
@@ -12,9 +12,9 @@
  * register view
  */
 typedef struct registers_t {
-	char name[32]; /*! The Name of the current register */
+	char name[32];   /*! The Name of the current register */
 	uint64_t offset; /*! Offset in the data block */
-	uint64_t size;	/*! Size of the register */
+	uint64_t size;   /*! Size of the register */
```
Comment (radare) [inline shlr/qnx/include/arch.h]: use ut64 for consistency instead of uint64_t

File: shlr/qnx/src/utils.c [hunk 2]
```diff
@@ -182,8 +185,7 @@ extract_unsigned_integer (const uint8_t *addr, int len, int be) {
 	return retval;
 }
 
-int
-i386nto_regset_id (int regno) {
+int i386nto_regset_id (int regno) {
```
Comment (radare) [inline shlr/qnx/src/utils.c]: can we `static` some of those functions?

File: shlr/qnx/src/utils.c [hunk 3]
```diff
@@ -182,8 +185,7 @@ extract_unsigned_integer (const uint8_t *addr, int len, int be) {
...
 	if (regno == -1)
 		return NTO_REG_END;
 	else if (regno < I386_NUM_GREGS)
```
Comment (radare) [inline shlr/qnx/src/utils.c]: if prev statement is a return, there's no need to else

File: shlr/qnx/src/utils.c (hunk 2 above)
Comment (madprogrammer) [inline shlr/qnx/src/utils.c]: these are kinda big and were pulled from `gdb`, so i'd prefer not to clutter the `core.c` file with it, additionally there soon may appear similar functions for ARM32 arch

**Comment by radare** [comment]
Looks good!

**Pull request closed without merging**